src/static/riot/competitions/detail/_header.tag

```diff
@@ -216,6 +225,14 @@
             $('.send-pop').popup('toggle')
         }
 
+        self.get_current_server_time = function () {
+            CODALAB.api.get_current_server_time()
```

Comment:
I think instead of doing a whole API call to get this, we can put current server time in `base.html` maybe? 

Can use the [now template tag.](https://docs.djangoproject.com/en/dev/ref/templates/builtins/?from=olddocs#now) 

```javascript
// I think this should work, "now()" shouldn't ever give us back a double quote
let CURRENT_DATE_TIME = "{% now "jS F Y H:i" %}"
```

Nice job implementing this pattern, though!

Reply:
It might even be nicer to return some isoformat date that way we can have a more robust Date object instead of a string. I forget if we have arrow/moment/whatever JS datetime lib..

Reply:
✅

Reply:
@brendon-lind @dde6khkg After addressing a comment, at the very least, we place a checkmark (***after implementing the change***) like the one above, just so the reviewer knows that it has been addressed.

Reply:
> It might even be nicer to return some isoformat date that way we can have a more robust Date object instead of a string. I forget if we have arrow/moment/whatever JS datetime lib..

Done

Reply:
Done!